### PR TITLE
Convert project to swift/kotlin

### PR DESCRIPTION
**Pull request by wesbillman**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -1,14 +1,16 @@
-group 'com.compass.storage'
+group 'com.example.storage'
```
Comment (russellbstephens) [inline android/build.gradle]: do you want to change bundles here?

File: example/test/widget_test.dart [hunk 2]
```diff
@@ -1,11 +1,27 @@
-//  Copyright © 2019 Compass. All rights reserved.
```
Comment (russellbstephens) [inline example/test/widget_test.dart]: should we add this back?

File: android/build.gradle (hunk 1 above)
Comment (wesbillman) [reply android/build.gradle]: I'll fix

File: example/test/widget_test.dart (hunk 2 above)
Comment (wesbillman) [reply example/test/widget_test.dart]: fixing, thanks

**Pull request merged**